examples/external-data/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # @fluid-example/app-integration-external-data
 
+## 2.22.0
+Replaced node-fetch with native fetch for improved compatibility and maintainability.
```

Comment:
These changelogs are automatically generated from changesets, so don't manually edit them.

Reply:
@tylerbutler I'm just realizing that we don't include a notice in these files that they are auto-generated. Would that be easy to add?